language/vm/vm_genesis/src/lib.rs

```diff
@@ -327,6 +327,10 @@ pub fn encode_genesis_transaction_with_validator(
 
             let mut txn_executor = TransactionExecutor::new(&block_cache, &data_cache, txn_data);
             txn_executor.create_account(genesis_addr).unwrap().unwrap();
+            txn_executor
+                .create_account(account_config::core_code_address())
```

Comment:
Bit concerned about the auth key for these accounts, but not required for this PR